test/release/examples/benchmarks/hpcc/ra.chpl

```diff
@@ -153,7 +158,9 @@ proc main() {
     forall (_, r) in zip(Updates, RAStream()) do
       T[r & indexMask] ^= r;
 
-  const execTime = getCurrentTime() - startTime;   // capture the elapsed time
+  kernel.stop();
```

Comment:
Why make RA the only of the HPCC benchmarks to use timers rather than getCurrentTime() without discussion?
Why stop kernel before reading the elapsed time rather than just reading the elapsed time?